glide.yaml

```diff
@@ -55,3 +55,5 @@ import:
 - package: github.com/go-openapi/validate
 - package: github.com/go-openapi/loads
 - package: github.com/elazarl/go-bindata-assetfs
+- package: github.com/dgraph-io/badger
```

Comment:
It needs `glide update` to update the lock file with this change.

Reply:
Sadly I'm not sure what's the proper way to update this project's dependencies. glide up will break Jaeger (for example viper is updated and will no longer compile) as that has no version locked in the glide.yaml

It also updates several other dependencies.

Reply:
What I did in a previous PR was to apply this change in a separate branch and then manually update the lock file *only* for the dependency I'm updating. Sucks, but works.